### DISCUSSION TITLE
Пример технического задания

### DISCUSSION THREAD

**Discussion by Dschaymaa** [vpo-tusur/todo-api#16] [Аналитика]
Залетайте, помогайте, исправляйте
https://docs.google.com/document/d/14TPqjCJLH7CqhjhXMNF-rv_4GInxWZq53POCNToSilE/edit?usp=sharing

**Comment by Dschaymaa** [comment]
Разве мы не под готовое приложение подстраиваемся? Или разработчики готовые поменять продукт под требования?

**Reply by asmazovec** [reply]
не понял тебя
